tensorboard/plugins/profile/pod_viewer/pod_viewer_dashboard/pod-viewer-dashboard.ts

```diff
@@ -30,19 +44,28 @@ Polymer({
     selectedChannel: {
       type: Array,
       notify: true,
-      observer: 'selectedChannelChanged_',
+      observer: '_selectedChannelChanged',
     },
-    activeBarChartEle: {
+    activeBar: {
       type: Object,
       notify: true,
-      observer: 'activeBarChartEleChanged_',
+      observer: '_activeBarChanged',
     },
     _error_message: {
       type: String,
     },
+    _podStatsMap: {
+      type: Object,
+      computed: '_getPodStatsMap(data)',
+      observer: '_podStatsMapChanged',
+    },
+    _runEnvironment: {
+      type: Object,
+      computed: '_getRunEnvironment(data)',
+    },
     _podStats: {
       type: Object,
-      observer: 'podStatsChanged_',
+      observer: '_podStatsChanged',
```

Comment:
Hmm I do not see any setter for this property. Can you make sure it is correct? Thanks!

Reply:
https://github.com/tensorflow/tensorboard/blob/cfbe444c781d2b67ed3f09877f8aa6ea62b298f3/tensorboard/plugins/profile/pod_viewer/pod_viewer_dashboard/pod-viewer-dashboard.ts#L158-L180

1. These lines need to change to `this._podStats = ...`
2. Please use computed properties.

Reply:
updated.